azure/Guidelines.md

```diff
@@ -846,110 +847,259 @@ https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#perfo
+
+#### LRO action on a resource pattern
+<a href="#post-or-delete-lro-pattern" name="post-or-delete-lro-pattern"></a><!-- Preserve old header link -->
+
+<a href="#lro-existing-resource" name="lro-existing-resource">:white_check_mark:</a> **DO** use the following pattern when implementing an LRO action operating on an existing resource:
+
+```text
+POST /UrlToExistingResource:<action>?api-version=<api-version>
+operation-id: <optionalStatusMonitorResourceId>`
+
+<JSON Action parameters in body>
+```
+
+The response must look like this:
+
+```text
+202 Accepted
+operation-id: <statusMonitorResourceId>
+operation-location: https://operations/<operation-id>
+
+<JSON Status Monitor Resource in body>
+```
+
+The request body contains information to be used to execute the action.
+
+For an idempotent POST (same `operation-id` and request body within some short time window), the service should return the same response as the initial request.
+
+For a non-idempotent POST, the service can treat the POST operation as idempotent (if performed within a short time window) or can treat the POST operation as initiating a brand new LRO action operation.
+
+<a href="#lro-no-post-create" name="lro-no-post-create">:no_entry:</a> **DO NOT** use a long-running POST to create a resource -- use PUT as described above.
+
+<a href="#lro-operation-id-request-header" name="lro-operation-id-request-header">:white_check_mark:</a> **DO** allow the client to pass an `Operation-Id` header with an ID for the operation's status monitor.
+
+<a href="#lro-operation-id-default-is-guid" name="lro-operation-id-default-is-guid">:white_check_mark:</a> **DO** generate an ID (typically a GUID) for the status monitor if the `Operation-Id` header was not passed by the client.
+
+<a href="#lro-operation-id-unique-except-retries" name="lro-operation-id-unique-except-retries">:white_check_mark:</a> **DO** fail a request with a `400-BadRequest` if the `Operation-Id` header matches an existing operation unless the request is identical to the prior request (a retry scenario).
+
+<a href="#lro-returns-202" name="lro-returns-202">:white_check_mark:</a> **DO** return a `202-Accepted` status code from the request that initiates an LRO if the processing of the operation was successfully initiated (except for "PUT with additional processing" type LRO).
+
+<a href="#lro-returns-only-202" name="lro-returns-only-202">:warning:</a> **YOU SHOULD NOT** return any other `2xx` status code from the initial request of an LRO -- return `202-Accepted` and a status monitor even if processing was completed before the initiating request returns.
+
+<a href="#lro-returns-status-monitor" name="lro-returns-status-monitor">:white_check_mark:</a> **DO** return a status monitor in the response body as described in [Obtaining status and results of long-running operations](#obtaining-status-and-results-of-long-running-operations).
+
+#### LRO action with no related resource pattern
```

Comment:
Not related to the guideline.

We may want to add a new operation template to Azure.Core typespec.
At current Azure.Core, dev would tend to use `LongRunningResourceCreateOrReplace` template, which is not fit for this "no related resource pattern".

Reply:
Requested review from TypeSpec team members.

Reply:
Opened an issue to typespec-azure-core to see if we are going to add a template https://github.com/Azure/typespec-azure/issues/787